server/datastore/mysql/migrations/tables/20241209164540_AddSecretVariablesTable.go

```diff
@@ -0,0 +1,33 @@
+package tables
+
+import (
+	"database/sql"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20241209164540, Down_20241209164540)
+}
+
+func Up_20241209164540(tx *sql.Tx) error {
+
+	if tableExists(tx, "secret_variables") {
+		return nil
+	}
+
+	_, err := tx.Exec(`
+		CREATE TABLE secret_variables (
+		id INT UNSIGNED NOT NULL AUTO_INCREMENT,
+		name VARCHAR(255) NOT NULL,
+		value BLOB NOT NULL,
```

Comment:
Double-checking that it's intentional - `BLOB` is 64KB (vs MEDIUM/LONG/TINY). It makes sense to me as a reasonable size vs too large or too small (tiny is 255b).

Reply:
yes, intentional